README.md

```diff
@@ -240,6 +258,27 @@ var registeredUsers = result.ElementAt(0);
 var registeredAndSubscribedUserCount = result.ElementAt(1);
 ```
 
+Timeframes
+------------
+
+A timeframe can be specified for analysis using the `QueryRelativeTimeframe` and `QueryAbsoluteTimeframe` classes, along with an optional `timezone` parameter passed to the `KeenClient.Query` method when using `QueryRelativeTimeframe`. The `timezone` parameter must be one of the timezones supported by the Keen.io API as specified [here](https://keen.io/docs/api/#timezone).
```

Comment:
I'll actually change this to Keen IO, since that seems to be how it's documented elsewhere.